### ISSUE TITLE
Grid in dialogs scrolls to the top on attempt to open a context menu

### ISSUE THREAD

**Issue by RoProg** [vaadin/flow-components#5376]
### Description

I opened a vaadin dialog with a grid inside and scrolled to the bottom. After right click on the grid (the grid context menu was triggered) the grid scrolled to the top.

### Expected outcome

The scroll position should not be changed by the context menu.

### Minimal reproducible example

```java
public class Item {
	public String getName() {
	    return "Test";
	}
	public int getAge() {
	    return 42;
	}
 }
...
final var dialog = new Dialog();

var grid = new Grid<>(Item.class);
grid.addContextMenu().add(new NativeLabel("Action"));

final var items = new Item[100];
Arrays.fill(items, new Item());
grid.setItems(items);

dialog.add(grid);
grid.setSizeFull();
dialog.setSizeFull();
dialog.open();
```

### Steps to reproduce

1. Run the code above.
2. Scroll to the bottom of the grid and then trigger the context menu (right mouse click).

### Environment

- Vaadin / Flow version: 24.1.4
- Java version: 17.0.1
- OS version: Windows 11
- Application Server (if applicable): tomcat 10.1

### Browsers

Chrome, Firefox

**Comment by RoProg** [comment]
I can confirm that, it is no longer reproducible, at least in Vaadin 24.2.2.

**Comment by web-padawan** [comment]
Thanks for confirmation. The issue is also fixed in 24.1.14 (although can be reproduced in earlier 24.1 versions). Closing.
